src/atom-environment.coffee

```diff
@@ -891,10 +891,8 @@ class AtomEnvironment extends Model
     @commands.dispatch(@contextMenu.activeElement, command, args)
 
   openLocations: (locations) ->
-    needsProjectPaths = @project?.getPaths().length is 0
-
     for {pathToOpen, initialLine, initialColumn} in locations
-      if pathToOpen? and needsProjectPaths
+      if pathToOpen?
```

Comment:
I think the reason that this guard was here is for this case:

```sh
$ atom directory-1 
# opens directory-1 in a new window

$ atom directory-2/file-2
# opens file-2 in the same window, but *does not* add directory-2 to the project
```

Does that still work? In other words, when I open a single file from the command line, does it still add it to the most recently focused Atom window, but *not* add the parent directory to the project?